charts/rancher-logging/0.0.1/charts/fluentd/values.yaml

```diff
@@ -0,0 +1,444 @@
+labels: {}
+
+image:
+  repository: micheliac/fluentd
+  tag: withoutconfig
+  pullPolicy: IfNotPresent
+# initImage:
+#   repository: rancher/fluentd
+#   tag: v0.1.11
+#   pullPolicy: IfNotPresent
+
+resources: {}
+  # limits:
+  #   cpu: 100m
+  #   memory: 500Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 200Mi
+env:
```

Comment:
Give a zero value to none defined field or comment it.

Reply:
comment it